File: Modul 6/SOAL-5/PRAK605_2410817220015_Ashma_Salimah.py
ordo = int(input(""))
matriksA = [[0] * ordo for _ in range(ordo)]
matriksB = [[0] * ordo for _ in range(ordo)]
hasil = [[0] * ordo for _ in range(ordo)]
print("Matriks A")
for i in range (ordo) :
    matriksA[i] = list(map(int,input("").split()))
print("Matriks B")
for i in range (ordo) :
    matriksB[i] = list(map(int,input("").split()))
for i in range (ordo) :
    for j in range (ordo) :
        hasil[i][j] = 0
        for k in range (ordo) :
            hasil[i][j] += matriksA[i][k] * matriksB[k][j]
print("\nMatriks AXB")
for i in range (ordo) :
    for j in range (ordo) :
        print(hasil[i][j], end=" ")
    print("")

File: Modul 6/SOAL-5/PRAK605_2410817220015_Ashma_Salimah.c
#include <stdio.h>
int main() {
    int ordo;
    scanf("%d", &ordo);
    int matriksA[ordo][ordo], matriksB[ordo][ordo], i, j;
    printf("Matriks A \n");
    for (i = 0; i < ordo; i++) {
        for(j = 0; j < ordo; j++) {
            scanf("%d", &matriksA[i][j]); } }
    printf("Matriks B \n");
    for (i = 0; i < ordo; i++) {
        for (j = 0; j < ordo; j++) {
            scanf("%d", &matriksB[i][j]); } }
    int hasil[ordo][ordo];
    printf("\nMatriks AXB\n");
    for (i = 0; i < ordo; i++) {
        for (j = 0; j < ordo; j++) {
            hasil[i][j] = 0;
            for (int k = 0; k < ordo; k++) {
                hasil[i][j] += matriksA[i][k] * matriksB[k][j]; } } }
    for (i = 0; i < ordo; i++) {
        for (j = 0; j < ordo; j++) {
            printf("%d ", hasil[i][j]); } 
        printf("\n");}
    return 0;
}